patroni/dcs/raft.py

```diff
@@ -121,7 +121,8 @@ def __init__(self, on_ready, on_set, on_delete, **config):
                            journalFile=(file_template + '.journal' if self_addr else None),
                            onReady=on_ready, dynamicMembershipChange=True)
 
-        super(KVStoreTTL, self).__init__(self_addr, partner_addrs, conf)
+        retry_timeout = int(config.get('retry_timeout') or 10)
```

Comment:
Can `retry_timeout` be explicitly set to 0? We want it to be 10 then?

Reply:
Am also wondering: don't we want this code-wide default timeout = 10 be set via some variable?